packages/luna/src/components/Chat/ChatHeader/ChatHeader.tsx

```diff
@@ -12,7 +12,7 @@ export const ChatHeader: React.FC<{
   name: string | null;
   image: string | null;
   online: boolean;
-  lastSeen: string;
+  lastSeen?: string;
```

Comment:
You can get this info from the backend (the same you did with name), no need to make it optional.